src/index.js

```diff
@@ -242,8 +242,11 @@ export default function buildPreset(context, opts = {}) {
     ]),
   );
 
-  useBuiltIns &&
-    plugins.push([transformPolyfillRequirePlugin, { polyfills, regenerator }]);
+  if (useBuiltIns === true) {
+    plugins.push([addUsedBuiltInsPlugin, { polyfills, regenerator, debug }]);
```

Comment:
Maybe we don't want to have those warnings, not sure if it's useful or not but we can iterate